lib/cli/src/frameworks/angular/1-Button.stories.ts

```diff
@@ -24,7 +24,18 @@ Emoji.args = {
   text: '😀 😎 👍 💯',
 };
 
-Emoji.parameters = { notes: 'My notes on a button with emojis' };
+export const WithCustomBackground = Template.bind({});
+WithCustomBackground.args = {
+  text: 'Defined via addon-backgrounds!',
+};
+
+WithCustomBackground.storyName = 'With custom background';
```

Comment:
why?

Reply:
good question, will remove it

Reply:
@shilman would you like me to remove the same for `ButtonWithLinkToAnotherStory` ?